src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  background-color: silver;
+}
+
+#input {
+  padding: 12px;
+  padding-right: 3rem;
```

Comment:
- [ ] The list structure should follow the recommended design. Please ensure that each item in the list has a checkbox and there is a `Clear all completed` element. on the page.

| Original | Yours |
| ----- | ------ |
| <img src='https://camo.githubusercontent.com/d04bee7a5b8f62cc6655dbec65c4d14583e4d78d0c434ddc070f6881bb9568fc/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067' height='250px'> | <img src='https://user-images.githubusercontent.com/33323077/130770267-c61fdd32-252b-4cf8-b22c-702ad98c7054.png' height='250px'> |